### DISCUSSION TITLE
실무에서 플러그인 써보셨거나 쓰는 사례 본 적 있으신가요?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#14] [Q&A]
저는 사실 플러그인 기능을 처음 봤는데요.
어쩌면 알게 모르게 기본적으로 쓰이고 있는 플러그인이 있을 수도 있겠지만,
인지하고 쓰는 케이스가 있으셨는지 궁금합니다!

**Comment by menthamin** [comment]
아마도 전문텍스트 기능일 것 같습니다
저도 제가 별도로 설치해본적은 없습니다 ㅎㅎ;
